allennlp/training/initializers.py

```diff
@@ -114,4 +111,8 @@ def curried_initializer(tensor: torch.Tensor):
                     return initializers[initializer_type](tensor, **initializer_params)
                     # pylint: enable=cell-var-from-loop
                 instantiated_initializers[name] = curried_initializer
-        return InitializerApplicator(instantiated_initializers)
+        try:
+            default = instantiated_initializers.pop("default")
```

Comment:
Can't you just do `default = instantiated_initializers.pop("default", torch.nn.init.normal)`?